tests/unit/hive_metastore/test_table_migrate.py

```diff
@@ -64,6 +66,7 @@ def test_migrate_dbfs_root_tables_should_produce_proper_queries(ws):
         group_manager,
         migration_status_refresher,
         principal_grants,
+        hiveserde_in_place_migrate=False,
```

Comment:
if the default value is False, do we still need this change?

Reply:
refactored this code